### PR TITLE
doc: adding hackweek proposal

### PR DESCRIPTION
**Pull request by asettle**
Signed-off-by: Alexandra Settle <asettle@suse.com>

### CODE REVIEW

**Review by jhesketh — DISMISSED** [review]
Cool, lgtm!

**Review by jecluis — APPROVED** [review]

**Review by ml8mr — DISMISSED** [review]

File: doc/spec/hackweek-proposal.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+Literally, anyone. We're looking for people who are interested in hacking on new projects, or want to learn something new. We're also more than happy to teach *you* about storage if you come from a different knowledge background.
+
+If you consider yourself to be non-technical, but interested in how communities work - we're also looking for people to get involved with the other sides of community management that are not related to code only.
+
+[comment]: # (What are your goals to be achieved at the end of this Hackweek?)
+
+## Goal for this Hackweek
+
+We are currently in Phrase 2 (Milestone 2) of the project where we are actively developing how to deploy a multi-node setup. Our goals:
```
Comment (tserong) [inline doc/spec/hackweek-proposal.md:27]: s/Phrase/Phase/

File: doc/spec/hackweek-proposal.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+- Testing deployment
+- Bug finding/squashing
+- And if you like the project, active development! We have a series of issues related to multi-node deployment, and anything you can grab is yours.
+
+If you consider yourself non-technical, but want to get involved another way:
```
Comment (tserong) [inline doc/spec/hackweek-proposal.md:33]: Can we say something other than "non-technical" here, or even just strip this back to "If you want to get involved in another way"?

**Review by tserong — CHANGES_REQUESTED** [review]
A couple of little nits, otherwise looks excellent

**Review by tserong — APPROVED** [review]

**Pull request merged**